src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resMass = [];
```

Comment:
it's not a good practice to name variables like this since it is not self-descriptive.
It's better to use something like result or resultArray or stateVersions or clonesArray etc.